src/BizHawk.Emulation.Common/Base Implementations/Bk2MnemonicLookup.cs

```diff
@@ -83,6 +83,11 @@ public static string LookupAxis(string button, string systemId)
 			["Run"] = 'R',
 			["RUN"] = 'R',
 
+			["A"] = 'A',
+			["B"] = 'B',
+			["X"] = 'X',
+			["Y"] = 'Y',
```

Comment:
Those should be unnecessary as the mnemonic fallback is already the string itself.